activemodel/lib/active_model/model.rb

```diff
@@ -61,6 +61,7 @@ module Model
     include ActiveModel::AttributeAssignment
```

Comment:
Let's put the Attributes include above here. All the other modules can technically expand on `attribute` so it makes sense that that's the top-level one (and why we want to include it by default.

Reply:
Thank you :)